### PR TITLE
Added Graduation Details

### PR DESCRIPTION
**Pull request by CSElonewolf**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/CSElonewolf/CSElonewolf.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Devkant Gorai 
+institution: Government College of Engineering and Textile Technology,Berhampore
```
Comment (Mohammed-Shoaib) [inline _data/CSElonewolf/CSElonewolf.md:3]: The name of the institution must **not** be longer than 58 characters, yours is **67** characters.

**Comment by CSElonewolf** [comment]
@Mohammed-Shoaib I have made the requested changes

File: _data/CSElonewolf/CSElonewolf.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Devkant Gorai 
+institution: Government College of Engineering and Textile Technology
+profile_pic: CSElonewolf.jpg
```
Comment (Mohammed-Shoaib) [inline _data/CSElonewolf/CSElonewolf.md:4]: Incorrect file name, change `CSElonewolf.jpg` to `CSELonewolf.jpg` or rename the image to `CSElonewolf.jpg`

**Comment by CSElonewolf** [comment]
@Mohammed-Shoaib made the requested changes

**Comment by elisemoe** [comment]
Hi @CSElonewolf 👋😄 we can't wait to celebrate graduation with you! Be sure to submit the form before 12:00 AM PT today (in ~2.5hr) to be considered for swag.

**Pull request merged**